hudi-sync/hudi-hive-sync/pom.xml

```diff
@@ -43,6 +45,11 @@
       <artifactId>hudi-hadoop-mr</artifactId>
       <version>${project.version}</version>
     </dependency>
+    <dependency>
+      <groupId>org.apache.hudi</groupId>
+      <artifactId>hudi-sync-common</artifactId>
```

Comment:
Thanks, I thank put hudi-sync-common base class to hudi-common is make sense. but hudi-sync-hive 、hudi-sync-dla under hudi/     directory  will make hudi code so much moudle .  
I think another choice to  put hudi-sync-common base class to hudi-common , and hudi-sync-hive 、hudi-sync-dla  etc.. under hudi-sync.
what about yours suggestion @vinothchandar @leesf ?

Reply:
hi, I am incline to put hudi-sync-common, hudi-sync-dla and hudi-sync-hive modules under hudi-sync, it will keep the structure same as the refactored hudi-client which is consisted of hudi-client-common, hudi-client-spark, hudi-client-flink, hudi-client-java modules.

```
hudi-sync
|--hudi-sync-common
|--hudi-sync-hive
|--hive-sync-dla


hudi-client
|--hudi-client-common
|--hudi-client-java
|--hudi-client-spark
|--hudi-client-flink
```

Reply:
thanks, be consistent with hudi-client will be better